deadpool.py

```diff
@@ -212,7 +302,7 @@ def conn_send_safe(obj):
             with lock:
                 conn.send(obj)
         except BrokenPipeError:  # pragma: no cover
-            logger.error("Pipe not usable")
+            logger.debug("Pipe not usable")
```

Comment:
Debug?

Reply:
Yeah, because it's pretty annoying to see these logs, and they will often happen when timeouts are applied, and finally, the user cannot do anything about them.